### PR TITLE
Slim install

### PR DESCRIPTION
**Pull request by david-deboer**
This distills the packages needed for install to various options for slimmer installations.

### CODE REVIEW

**Comment by bhazelton** [comment]
@mkolopanis can you review this? I wrote as much or more of it than Dave did, so outside eyes would be good.

File: hera_mc/cm_dossier.py [hunk 1]
```diff
@@ -187,6 +186,7 @@ def table_row(self, columns, ports):
         list
             A row or rows for the tabulate display.
         """
+        from itertools import zip_longest
```
Comment (mkolopanis) [inline hera_mc/cm_dossier.py:189]: I'm a little confused here. Isn't iterools in the standard libraray? I suppose it doesn't matter too much, but just feels weird to move the import.

**Review by mkolopanis — COMMENTED** [review]
I think this is all pretty good. I have one little comment about a standard lib package.

File: hera_mc/cm_dossier.py (hunk 1 above)
Comment (bhazelton) [reply hera_mc/cm_dossier.py:189]: I agree, I think it should be moved back @david-deboer

**Review by mkolopanis — APPROVED** [review]
Looks good thanks for the adjustment.

**Pull request merged**